sys/vane/ames.hoon

```diff
@@ -432,7 +432,7 @@
           [& $(liv l.liv)]
       ?~  ack  [~ ~ liv]
       =.  ded  ?:(top [n.liv ded] ded)
-      =.  ded  ?:(vig.clu.u.ack (~(tap to r.liv) ded) ded)
+      =?  ded  vig.clu.u.ack  (weld ~(tap to r.liv) ded)
```

Comment:
This is the one call site (along with a similar expression in `++flay` that had to be actually refactored (where I couldn't just remove parenthesis). The same expressions are repeated in`xmas.hoon`. `+-tap` used to prepend to its argument, so this isn't exactly a complex change. I just wanted to call it out specifically. I haven't tested the expression in a generator, but I haven't tested network operations ...